src/App.tsx

```diff
@@ -33,25 +40,62 @@ export const PATHS = {
   WELCOME_PAGE: "/welcome",
 };
 
+const queryClient = new QueryClient({
+  defaultOptions: {
+    mutations: {
+      networkMode: 'online',
+    },
+    queries: {
+      networkMode: 'online',
+      staleTime: 1000 * 20, // 20 seconds
+      gcTime: 1000 * 60 * 60 * 24 * 7, // 1 week
+      retry: 0,
+    }
+  }
+});
+addDefaultMutationFns(queryClient);
+
+const store = new Storage();
```

Comment:
The [README.md file](https://github.com/ionic-team/ionic-storage) of `@ionic/storage` says:

> A simple key-value Storage module for Ionic apps. This utility uses the best storage engine available on the platform without having to interact with it directly (some configuration required, see docs below).